TLSharp.Core/TelegramClient.cs

```diff
@@ -187,12 +187,20 @@ public bool IsUserAuthorized()
 
             return ((TLUser)request.Response.user);
         }
+        
         public async Task<TLPassword> GetPasswordSetting()
         {
             var request = new TLRequestGetPassword();
 
-            await _sender.Send(request);
-            await _sender.Receive(request);
+            try
```

Comment:
actually these try-catches you added are not correct; they need a while loop to work; please see how it's done in other similar try-catch blocks.

Reply:
No worries, can you now squash your two commits into one please? Easiest way to do it is using `git rebase -i HEAD~2`